app/controllers/place_controller.rb

```diff
@@ -27,13 +27,17 @@ def publication
   end
 
   def locals
+    locals = {
+      results_anchor: "results",
+    }
+
     if params[:slug] == REPORT_CHILD_ABUSE_SLUG
-      {
+      locals.merge({
         option_partial: "option_report_child_abuse",
         preposition: "for",
-      }
+      })
     else
-      {}
+      locals
```

Comment:
Thanks for making this more generic. I'm sure that there are other services that will benefit from this anchor.

A minor point: you don't need the `else`, you can just return `locals` in the last line of the method. This may mean the `merge` will need to change to a `merge!` though.

Reply:
ah yeah that makes more sense!